clio-client/src/test/scala/org/broadinstitute/clio/client/webclient/ClioWebClientSpec.scala

```diff
@@ -91,6 +92,6 @@ class ClioWebClientSpec
   }
 
   it should "retry requests that fail with connection errors" in {
-    client.getSchema(WgsUbamIndex).map(_ should be(WgsUbamIndex.jsonSchema))
+    client.query(index)(ModelMockQueryInput(), includeDeleted = false).map(_ => succeed)
```

Comment:
How does this test retry on connection error? I think I'm missing something

Reply:
The mock server is set up to time out this specific route N-1 times... I'll add a comment.